nalgebra-sparse/src/pattern.rs

```diff
@@ -184,6 +184,32 @@ impl SparsityPattern {
         })
     }
 
+    /// Try to construct a sparsity pattern from the given dimensions, major offsets
+    /// and minor indices.
+    ///
+    /// Returns an error if the data does not conform to the requirements.
```

Comment:
Docs here are not correct: it does not do any checking except panicking if the major offsets have the wrong size. Ideally this should be documented in a `# Panics` section.

Reply:
I removed the # Errors section, as no errors are returned here.